kofu/build.gradle.kts

```diff
@@ -50,6 +50,7 @@ dependencies {
 	testCompile("org.testcontainers:testcontainers:1.11.3")
 	testImplementation("redis.clients:jedis")
 	testImplementation("io.lettuce:lettuce-core")
+	testImplementation("org.mongodb:mongodb-driver-legacy")
```

Comment:
Could you please elaborate on this `mongodb-driver-legacy` dependency?

Reply:
in order to run the test, `MongoInitializer` needs `com.mongodb.client.MongoClient` from that dependency.